.pylintrc

```diff
@@ -119,7 +119,8 @@ spelling-store-unknown-words=no
 [MISCELLANEOUS]
 
 # List of note tags to take in consideration, separated by a comma.
-notes=FIXME,XXX,TODO
+# notes=FIXME,XXX,TODO
+notes=FIXME,XXX
```

Comment:
Can we revert this change? It doesn't look like you're adding any TODO comments and we're trying to avoid diffs from the cookiecutter generated code where possible.

As a fortuitous second effect, it should trigger the now merged actions build trigger to fire so we can check that this PR will work ok with the new process

Reply:
Reverted.